docs/index.md

```diff
@@ -100,7 +100,7 @@ This will only install your package in *editable* mode and the minimum
 dependencies required to run `nox`. It will then run all `nox` default
 sessions, which include running linters and tests.
 
-!!! note
+!!! Tip
```

Comment:
`!!! Note`

Reply:
Oh, that was changed on purpose, is more a tip than a note, forgot to mention it in the commit